bodhi/tests/server/services/test_updates.py

```diff
@@ -3624,12 +3623,8 @@ def test_stable_request_when_stable(self, publish, *args):
 
         self.assertEqual(resp.json['update']['status'], 'stable')
         self.assertEqual(resp.json['update']['request'], None)
-        try:
-            publish.assert_called_with(
-                topic='update.request.stable', msg=mock.ANY)
-            assert False, "request.stable fedmsg shouldn't have fired"
-        except AssertionError:
-            pass
```

Comment:
Hahahahaha, whaaaaaat was going on there?

Reply:
Yeah, there are some places like this one in Bodhi. Take a look at `validate_updateid` :D